DataHubClient.js

```diff
@@ -290,12 +298,14 @@ export class DataHubClient {
     }
 
     // update indexed entries and jwe
+    const {meta, content} = doc;
```

Comment:
We should keep these in order for consistency:

```suggestion
    const {content, meta} = doc;
```